config/rbac/vg_manager_cluster_role.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Nit. Please rename the file to vg_manager_clusterrole.yaml

Reply:
done